presto-docs/src/main/sphinx/release/release-0.238.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.238
+=============
+
+**Highlights**
+==============
+* Fix SQL function parameters to be case-insensitive.
+* Add support to create external function. (:issue:`13254`)
+* Add support for exchange materialization of table bucketed by non-hive types. This can be enabled by setting the ``bucket_function_type_for_exchange`` session property or ``hive.bucket-function-type-for-exchange `` configuration property to ``PRESTO_NATIVE``
+* Add predicate pushdown support for ``DATE``, ``TIMESTAMP``, and ``TIMESTAMP_WITH_TIME_ZONE`` literals for Pinot.
+
+General Changes
+_______________
+* Fix case sensitivity issue in SQL function parameters. Function parameters should be treated case-insensitive.
```

Comment:
I prefer the highlight version of this - `Fix SQL function parameters to be case-insensitive.`